R/download_dataset.R

```diff
@@ -0,0 +1,170 @@
+#' Download data for publication
+#'
+#' ...
+#'
+#' @param connection A valid connection to the ETN database.
+#' @param animal_project_code (string) Animal project to download data from.
+#' @param directory (string) Path to local download directory.
+#' @param scientific_name (string) One or more scientific names to filter on.
+#'
+#' @return ...
+#'
+#' @export
+#'
+#' @importFrom glue glue
```

Comment:
We should also add:

```r
#' @importFrom readr write_csv
```

Reply:
Now added

Reply:
Confirmed. Thanks. You were faster than me.